lib/global.dart

```diff
@@ -37,11 +37,11 @@ String translatePlural(String key, num value, {Map<String, dynamic> args})
 	return Localization.instance.plural(key, value, args: args);
 }
 
-void changeLocale(BuildContext context, String localeCode)
+void changeLocale(BuildContext context, String localeCode) async
```

Comment:
```suggestion
Future changeLocale(BuildContext context, String localeCode) async
```
You should return a Future, if you make the method async.